packages/next/next-server/server/api-utils.ts

```diff
@@ -337,8 +339,8 @@ export function tryGetPreviewData(
       tokenPreviewData,
       options.previewModeSigningKey
     ) as typeof encryptedPreviewData
-  } catch {
-    // TODO: warn
+  } catch (e) {
+    console.warn(`Failed to veriry jsonwebtoken: ${e}`)
```

Comment:
This warning is not clear for end users.

Reply:
At this point, I do not have good idea of warning message for end users.
If end users does not jwt, navigate to URL of jwt spec.